report/2.Analysis/Requirements.tex

```diff
@@ -0,0 +1,39 @@
+\section{Requirements}
+In order to be able to conclude upon the success of the project, a fixed set of requirements must be formulated.
+These requirements will serve as a guideline in the conclusion of the report and as markers throughout the project. 
+To formally specify the requirements, the MoSCoW analysis model will be utilized. 
+The MoSCoW model specifies which requirements \textbf{M}ust, \textbf{S}hould, \textbf{C}ould, and \textbf{W}on't be fulfilled to deem the project successful.
+
+\textbf{Must have}
+\begin{itemize}
+\item ability to recognize a moving target in front of the device using an attached camera
+\item ability to hit a target moving with a constant speed along a horizontal axis
+\item move fast enough to keep the laser on the moving target in real time
+\item ability to move the laser at least 60 degrees in both horizontal and vertical directions from the origin.
+\item ability to predict the next location of the object moving along a linear pattern
+\item enough structural stability to be able to move without breaking
+\end{itemize}
+
+\textbf{Should have}
+\begin{itemize}
+\item ability to hit a larger moving target in the air at a low speed, such as a balloon \item ability to hit a small moving target in the air, such as red ball
+\item ability to predict the next location of the object moving along a non-linear path
+\end{itemize}
+
+\textbf{Could have}
+\begin{itemize}
+\item ability to calculate the distance to the target object
+\item use ballistic projectile to hit the target
+\item calculate the angle and power necessary to hit the target with a projectile
+\end{itemize}
+
+\textbf{Won't have at this time}
+\begin{itemize}
+\item ability to actually take down objectives
+\item ability to differentiate between the type of object that is being observed
+\end{itemize}
+
+By fulfilling the requirements specified in the must have section, the device will be able to track a target with a rather predictable movement pattern and use the attached laser to indicate the tracking. 
```

Comment:
`in the \textit{must have} section`
og ligeledes i resten af denne paragraf når der nævnes "could have" eller "would have" ' wont have' etc. om det er itallic eller typewriter eller hvad nu du bruger er lige fedt :P Men det er en konkret reference så der skal bindestreg eller anden ting til for at vise hvad du mener